modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,7 +3,151 @@
+    ├── mec-edge-sno1.yaml
+    ├── sno.yaml
+    └── source-crs <1>
+        ├── PaoCatalogSource.yaml
+        ├── PaoSubscription.yaml
+        ├── bell
+        |   ├── apiserver-config.yaml
+        |   └── disable-nic-lldp.yaml
+        └── elasticsearch
+            ├── ElasticsearchNS.yaml
+            └── ElasticsearchOperatorGroup.yaml
+----
+<1> The `sources-crs` subdirectory is in the same directory as `kustomization.yaml`.
+
+. Update the required `PolicyGenTemplate` CRs to include references to the content you added in the `custom-crs` directory.
++ 
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-dev" 
+  namespace: "ztp-clusters" 
+spec:
+  bindingRules:
+    dev: "true" 
+  mcp: "master" 
+  sourceFiles:
+    # These policies/CRs come from the internal container Image
+    #Cluster Logging
+    - fileName: ClusterLogNS.yaml
+      remediationAction: inform 
+      policyName: "group-dev-cluster-log-ns" 
+    - fileName: ClusterLogOperGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-cluster-log-operator-group" 
+    - fileName: ClusterLogSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-cluster-log-sub" 
+    #Local Storage Operator
+    - fileName: StorageNS.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-ns" 
+    - fileName: StorageOperGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-operator-group" 
+    - fileName: StorageSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-lso-sub" 
+    # Performance Addon Operator
+    - fileName: PaoSubscriptionNS.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-ns" 
+    - fileName: PaoSubscriptionCatalogSource.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-cat-source" 
+      spec:
+        image: <image URL here>
+    - fileName: PaoSubscription.yaml
+      remediationAction: inform 
+      policyName: "group-dev-pao-sub" 
+    #These are custom local polices that come from the source-crs directory in the git repo
+    #Elasticsearch Operator
+    - fileName: elasticsearch/ElasticsearchNS.yaml <1>
+      remediationAction: inform 
+      policyName: "group-dev-elasticsearch-ns" 
+    - fileName: elasticsearch/ElasticsearchOperatorGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-elasticsearch-operator-group" 
+    #Bell Custom Resources
+    - fileName: bell/apiserver-config.yaml
+      remediationAction: inform 
+      policyName: "group-dev-apiserver-config" 
+    - fileName: bell/disable-nic-lldp.yaml
+      remediationAction: inform 
+      policyName: "group-dev-disable-nic-lldp" 
+----
+<1> Set `fileName` to include the custom CR subfolder from the `/source-crs` parent.
+
+. Commit the `PolicyGenTemplate` change in Git, and then push to the Git repository being monitored by the GitOps ZTP Argo CD application.
```

Comment:
GitOps ZTP Argo CD application => GitOps ZTP ArgoCD policies application - Good to update this way.